docs/FAQ.md

```diff
@@ -307,11 +307,15 @@ Yes
 
 ### Are the R APIs for Spark supported?
 
-Yes, but we don't actively test them.
+Yes, but we don't actively test them. It is because the RAPIDS Accelerator hooks into Spark not at 
```

Comment:
Suggestion for this text and the Java API text below. 
```suggestion
Yes, but we don't actively test them, because the RAPIDS Accelerator hooks into Spark not at 
```
```suggestion
Yes, but we don't actively test them. It is because the RAPIDS Accelerator hooks into Spark not at 
```